srv/components/s3server/config/prepare.sls

```diff
@@ -15,25 +15,19 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+include:
+  - components.misc_pkgs.rsyslog.start
+  - components.ha.haproxy.start
+  - components.misc_pkgs.openldap.start
+
+Start sshd service:
+  service.running:
+    - name: sshd
+    - enable: True
+
 Stage - Prepare S3server:
   cmd.run:
     - name: __slot__:salt:setup_conf.conf_cmd('/opt/seagate/cortx/s3/conf/setup.yaml', 's3:prepare')
     - failhard: True
 
-# {% import_yaml 'components/defaults.yaml' as defaults %}
```

Comment:
Why are we removing the sections to add s3 repos?

Reply:
It was a part of EOS-18921 - Explode config files.
https://github.com/Seagate/cortx-prvsnr/pull/1070